From ad7a9a2d295c0525e0f4ba1b85045d160458e8b3 Mon Sep 17 00:00:00 2001
From: Manivannan Sadhasivam <manivannan.sadhasivam@oss.qualcomm.com>
Date: Wed, 10 Jun 2026 20:47:52 +0530
Subject: [PATCH 03386/15333] MAINTAINERS: Add Jeff Hugo as the Reviewer of MHI
 bus

Jeff has been active in reviewing the MHI patches and he volunteered to
become an official reviewer.

Signed-off-by: Manivannan Sadhasivam <manivannan.sadhasivam@oss.qualcomm.com>
Signed-off-by: Manivannan Sadhasivam <mani@kernel.org>
Link: https://patch.msgid.link/20260610151752.9373-1-manivannan.sadhasivam@oss.qualcomm.com
---
 MAINTAINERS | 1 +
 1 file changed, 1 insertion(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index 15011f5752a9..a4b89bd497ad 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -17463,6 +17463,7 @@ F:	arch/arm64/boot/dts/marvell/armada-3720-uDPU.*
 
 MHI BUS
 M:	Manivannan Sadhasivam <mani@kernel.org>
+R:	Jeff Hugo <jeff.hugo@oss.qualcomm.com>
 L:	mhi@lists.linux.dev
 L:	linux-arm-msm@vger.kernel.org
 S:	Maintained
-- 
2.55.0

